src/encoded/static/components/cart/batch_download.js

```diff
@@ -200,3 +208,147 @@ CartBatchDownload.contextTypes = {
 };
 
 export default CartBatchDownload;
+
+
+/**
+ * Displays the dropdown that allows the user to select a dataset type to download.
+ */
+const StaticBatchDownloadDatasetTypeSelector = ({
+    datasetTypes,
+    selectedDatasetType,
+    setSelectedDatasetType,
+}, reactContext) => {
+    const onChange = (event) => {
+        setSelectedDatasetType(event.target.value);
+    };
+
+    return (
+        <div className="cart__batch-download-dataset-type-selector">
+            <>Select download dataset type</>
+            <select value={selectedDatasetType} onChange={onChange}>
```

Comment:
I have another comment later on for another instance like this, but I was curious about why you have an onChange function and don't set the dataset type inline.

Reply:
👍